vm/isolated/isolated.go

```diff
@@ -173,26 +182,38 @@ func (inst *instance) ssh(command string) error {
 
 func (inst *instance) repair() error {
 	log.Logf(2, "isolated: trying to ssh")
-	if err := inst.waitForSSH(30 * time.Minute); err == nil {
+	if err := inst.waitForSSH(3 * time.Minute); err == nil {
```

Comment:
This changes logic. Previously we waited for ssh and the did the reboot to repair tainted kernel. Now we don't reboot. Please keep the old logic. We need to repair it.